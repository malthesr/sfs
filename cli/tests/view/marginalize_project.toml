args = [
  "view",
  "--precision",
  "2",
  "--marginalize-remove",
  "1",
  "--project-shape",
  "2,2",
  "tests/view/three_populations.sfs"
]
