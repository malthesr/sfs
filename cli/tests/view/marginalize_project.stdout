#SHAPE=<2/2>
42.75 51.75 123.75 132.75
